xml/art_sle_amazon_ecs.xml

```diff
@@ -41,22 +41,29 @@
     <para>
       Creating a new ECS cluster in Amazon is accomplished by using the
       <link xlink:href="https://console.aws.amazon.com/ecs/home#/firstRun">First
-      Run Wizard</link>
+      Run Wizard.</link> Alternatively you may use the
+      <command>aws</command> command line tool to create the cluster, refer to 
+      <link xlink:href=http://docs.aws.amazon.com/cli/latest/reference/ecs/create-cluster.html">create</link> for details.
+      In the First Run Wizard select
+      <guibutton>Custom</guibutton> if you want to create a
+      setup unique to your own needs. The
+      <guibutton>Amazon ECS sample</guibutton> selection creates
+      a pre-defined web application setup, that may be customized. By default
+      the First Run Wizard selects the Amazon ECS AMI, we will change this
+      later. 
+      <note>Note that using the First Run Wizard also creates an autoscaling
```

Comment:
This isn't a valid DocBook construct. I would recommend the following:

1. Close the paragraph after "later.":

    ```
... we will change this later.</para>
```

1. Open a new note:

   ```
<note>
    <!-- <title>...</title> --><!-- Optional title -->
    <para>Using the First Run  Wizard also creates ... </para>
</note>
```

1. Don't forget to remove the closing `</para>`. ;-)